data/theme.js

```diff
@@ -23,16 +23,16 @@ const colors = {
   red800: '#630000',
   red900: '#470000',
 
-  purple0: '#eeeaf1',
+  purple0: '#F6EAF7',
   purple100: '#dbd4e3',
   purple200: '#c6bad1',
-  purple300: '#ac9bbd',
+  purple300: '#BE87CC',
   purple400: '#8a71a2',
   purple500: '#4c2570',
-  purple600: '#432063',
+  purple600: '#6A008C',
   purple700: '#3a1c55',
-  purple800: '#2e1644',
-  purple900: '#1e0e2c',
+  purple800: '#3D0059',
+  purple900: '#1E0030',
```

Comment:
_UPPERCASE_? 😆

Reply:
That's how they come out of Sketch! But I manually fixed them.